examples/gitlab/README.md

```diff
@@ -0,0 +1,46 @@
+## GitLab Push EventListener
+
+Creates an EventListener that listens for Gitlab webhook events. To expose the
+EventListener, add a `serviceType: LoadBalancer`.
+
+### Try it out locally:
+
+1. Create the service account:
+
+```shell script
+kubectl apply -f examples/role-resources/triggerbinding-roles
+kubectl apply -f examples/role-resources/
+```
+
+1. Create the Gitlab EventListener:
+
+```shell script
+kubectl apply -f examples/gitlab/gitlab-push-listener.yaml
+```
+
+1. Port forward:
+
+```shell script
+kubectl port-forward \
+ "$(kubectl get pod --selector=eventlistener=gitlab-listener -oname)" \
+  8080
+```
+
+2. Test by sending the sample payload.
+
+```shell script
+curl -v \
+-H 'X-GitLab-Token: abcde' \
+-H 'X-Gitlab-Event: Push Hook' \
+-H 'Content-Type: application/json' \
```

Comment:
If you want to, we can put the Headers in a separate plaintext file:
```
X-GitLab-Token: abcde
X-Gitlab-Event: Push Hook
Content-Type: application/json
```
Then the curl command can look something like:
```bash
curl -v \
 -H @"examples/gitlab/gitlab-push-event-header" \
 --data-binary @"examples/gitlab/gitlab-push-event.json" \
 http://localhost:8080
```
which might be a little cleaner & easier to reproduce.

Reply:
Interesting idea...maybe we can come up with a convention each example can have a header.txt and a body.json or something...can be easily scripted that way!

Reply:
Yep 👍